doc/sphinx/user_manual/basic_starter_tutorial/01_input_design.md

```diff
@@ -2,4 +2,4 @@
 World Builder input file design
 =====================================
 
-GWB input files are plain text files using the JSON format. This means you can use any text editor to view and edit the files. If you are not familiar with the JSON format, it might be useful to familiarise yourself with it before continuing, although it is not necessary. The basic idea is that it allows data to be unambiguously structured to be readable by both humans and computers. 
+GWB input files are plain text files using the JSON format which allows data to be unambiguously structured to be readable by both humans and computers. You can use any text editor to view and edit the files. If you are not familiar with the JSON format, it might be useful to familiarise yourself with it before continuing, although it is not necessary. If you use Visual Studio Code, settings are included in the main repository (see /.vscode) to make editing World Builder files easier. 
```

Comment:
```suggestion
GWB input files are plain text files using the JSON format, which allows data to be unambiguously structured to be readable by both humans and computers. You can use any text editor to view and edit the files. If you are not familiar with the JSON format, it might be useful to familiarise yourself with it before continuing, although it is not necessary. If you use Visual Studio Code, settings are included in the main repository (see /.vscode) to make editing World Builder files easier. 

You may need to select "JSON with comments" as the file type to get proper syntax highlighting.
```